inc/search.class.php

```diff
@@ -3338,7 +3339,7 @@ static function addDefaultToView($itemtype, $params) {
 
       // Add entity view :
       if (Session::isMultiEntitiesMode()
-          && $item->isEntityAssign()
+          && $entity_check
```

Comment:
Fix bug introduced by #6966, the special case "AllAssets" is not a real object and can't be instantiated.